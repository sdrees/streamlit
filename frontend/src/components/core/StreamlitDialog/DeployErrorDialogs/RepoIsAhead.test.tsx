import RepoIsAhead from "./RepoIsAhead"

describe("DeployErrorDialog - RepoIsAhead", () => {
  it("Should render without crashing", () => {
    const dialog = RepoIsAhead()

    expect(dialog).toMatchSnapshot()
  })
})
