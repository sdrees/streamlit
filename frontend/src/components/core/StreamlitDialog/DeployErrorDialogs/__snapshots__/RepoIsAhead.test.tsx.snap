// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeployErrorDialog - RepoIsAhead Should render without crashing 1`] = `
Object {
  "body": <React.Fragment>
    <p>
      This Git repo has uncommitted changes. You may want to commit them before continuing.
    </p>
  </React.Fragment>,
  "title": "Unable to deploy app",
}
`;
